src/gollum/phoenix.py

```diff
@@ -387,18 +386,18 @@ def create_interact_ui(doc):
                 toolbar_location="below",
                 border_fill_color="whitesmoke",
             )
-            fig.title.offset = -10
-            fig.yaxis.axis_label = "Flux "
-            fig.xaxis.axis_label = "Wavelength (micron)"
+            fig.title.offset = 150
+            fig.yaxis.axis_label = "Flux"
+            fig.xaxis.axis_label = "Wavelength (\u00B5m)"
```

Comment:
Bokeh recently added support for LaTeX, so hypothetically we could add a `$\mu$` text here.  Let's look into that in the future, but for now, unicode makes sense.